libs/components/ag-grid/src/lib/modules/ag-grid/ag-grid.service.ts

```diff
@@ -322,6 +322,8 @@ export class SkyAgGridService implements OnDestroy {
           valueFormatter: autocompleteFormatter,
           comparator: autocompleteComparator,
           minWidth: 185,
+          autoHeight: false,
```

Comment:
do you have any concerns that this change in defaults might impact existing grids? Is it not set on the text cell type because that is the only one you anticipate actually needing to wrap/have automatic height?

Reply:
Good call, removed that change.